src/backend/joanie/tests/core/test_utils_contract_definition_generate_document_context.py

```diff
@@ -209,6 +210,7 @@ def test_utils_contract_definition_generate_document_context_without_order(self)
                     "first_name": "<ORGANIZATION_FIRST_NAME>",
                     "postcode": "<ORGANIZATION_ADDRESS_POSTCODE>",
                     "title": "<ORGANIZATION_ADDRESS_TITLE>",
+                    "is_main": False,
```

Comment:
Even if this attribute is useless, to stick to real case, you should set it to True.

Reply:
I didn't set it, it is the default model value.

Reply:
Oh right! Look's good to me like that.